bin/BuildPackages.sh

```diff
@@ -279,8 +248,8 @@ build_one_package() {
   ) || build_fail
 }
 
-date >> "$LOGDIR/$FAILPKGFILE.log"
-for PKG in "${PACKAGES[@]}"
+date >> "$LOGDIR/fail.log"
+for PKG in ${PACKAGES}
```

Comment:
I am fully aware of this, but package names are not allowed to contain spaces, and if any package dir ever contains a space, lots of other things will break. So all in all, I favor this well-known idiom over using bash arrays, which are in comparision an obscure feature.